dash/_cli.py

```diff
@@ -0,0 +1,171 @@
+import argparse
+import importlib
+import sys
+from typing import Any, Dict
+
+from dash import Dash
+
+
+def load_app(app_path: str) -> Dash:
+    """
+    Load a Dash app instance from a string like "module:variable".
+
+    :param app_path: The import path to the Dash app instance.
+    :return: The loaded Dash app instance.
+    """
+    app_split = app_path.split(":")
```

Comment:
check that `len(app_split) == 2` here?

Reply:
OK, I see that further down you're allowing users to provide just the module name, then using the first `Dash` you find in that module.